### PR TITLE
#1874-Sysdig Monitoring Team creation

### PR DESCRIPTION
**Pull request by guru-aot**
Sysdig monitoring team creation yml is created to run the agent in our namespace in openshift.

### CODE REVIEW

File: devops/openshift/sysdig-team.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      role: ROLE_TEAM_EDIT
+    - name: mohammed.alnahari@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: john.marchesan@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: harold.a.johnson@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: gurumoorthy.mohan@aot-technologies.com
+      role: ROLE_TEAM_EDIT
+    - name: dheepak.rr@aot-technologies.com
```
Comment (dheepak-aot) [inline devops/openshift/sysdig-team.yml:22]: shouldn't we use our gov email here?

Comment (guru-aot) [reply devops/openshift/sysdig-team.yml:22]: The login in controlled via github single sign on, the sysdig application verifies the email id configured in your github id. As we are using our github id created from aot, we have to use our ato technologies email.

**Review by ann-aot — APPROVED** [review]
👍

**Review by dheepak-aot — APPROVED** [review]
Thanks for all the work with sysdig 👍

**Review by sh16011993 — APPROVED** [review]
👍

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

File: devops/openshift/sysdig-team.yml (hunk 1 above)
Comment (JasonCTang) [reply devops/openshift/sysdig-team.yml:22]: For my clarification.  We're not sending alerts to everyone on this list right? ALl these people should have access, but not receive email alerts directly.  We will want to send the alerts to a generic email address.

Comment (guru-aot) [reply devops/openshift/sysdig-team.yml:22]: Regarding the alerts, there are none right now, we can create a Email notification channel where all the emails can go to a generic email address.

**Review by JasonCTang — APPROVED** [review]
Reviewed permissions and they are correct.

**Pull request merged**